<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="CAP-LAP-894" failures="1" tests="7" name="com.bryne.qa.testcases.UpdateDetailsPageTest" time="273.837" errors="0" timestamp="2021-07-06T11:03:43 IST" skipped="0">
  <testcase classname="com.bryne.qa.testcases.UpdateDetailsPageTest" name="updateBusinessPhoneTest" time="36.008"/>
  <system-out/>
  <testcase classname="com.bryne.qa.testcases.UpdateDetailsPageTest" name="updateFirstNameTest" time="42.495"/>
  <system-out/>
  <testcase classname="com.bryne.qa.testcases.UpdateDetailsPageTest" name="updateLastNameTest" time="33.534"/>
  <system-out/>
  <testcase classname="com.bryne.qa.testcases.UpdateDetailsPageTest" name="updateZipTest" time="42.045"/>
  <system-out/>
  <testcase classname="com.bryne.qa.testcases.UpdateDetailsPageTest" name="updateAddressTest" time="38.630">
    <failure message="expected [Sector 2 ,HSR Layout] but found []" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [Sector 2 ,HSR Layout] but found []
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at com.bryne.qa.testcases.UpdateDetailsPageTest.updateAddressTest(UpdateDetailsPageTest.java:92)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- updateAddressTest -->
  <system-out/>
  <testcase classname="com.bryne.qa.testcases.UpdateDetailsPageTest" name="updateStateTest" time="34.684"/>
  <system-out/>
  <testcase classname="com.bryne.qa.testcases.UpdateDetailsPageTest" name="updateCityTest" time="46.441"/>
  <system-out/>
</testsuite> <!-- com.bryne.qa.testcases.UpdateDetailsPageTest -->
